third-party/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ add_subdirectory(sml)
 add_subdirectory(taglib)
 add_subdirectory(tinyexpr)
 add_subdirectory(utz)
+#add_subdirectory(dr_libs)
```

Comment:
where is the cmake for dr_libs? is it included in the subrepo?

Reply:
fixed